app/models/spree/package_decorator.rb

```diff
@@ -7,7 +7,7 @@ def easypost_parcel
       end
 
       def use_easypost?
-        shipping_categories.any? { |shipping_category| shipping_category.use_easypost }
+        !order.ship_address.nil? && shipping_categories.any? { |shipping_category| shipping_category.use_easypost }
       end
 
```

Comment:
I would change this from `!order.ship_address.nil?` to `order.ship_address.present?`